### PR TITLE
Added old members management functionnalities

### PR DESCRIPTION
**Pull request by Louis-Bly**
You need to make migrations before working on the branch

Front is suboptimal, I let David take care of it

### CODE REVIEW

**Review by TB023 — DISMISSED** [review]

**Review by bastienlc — CHANGES_REQUESTED** [review]
The front needs to be fixed before merging.

**Comment by sachabinder** [comment]
We maybe should add some toggles to display other promo (this allow to choose if we want to see or not) as [here](https://enpc.org/clubinfo/clubinfo.enpc.org/equipe.html)

File: back/social/templates/social/club_edit.html [hunk 1]
```diff
@@ -90,52 +131,52 @@
...
+                                        <input class="button blue-button" type="submit" name="Vieux" value="Inactif">
+                                        <input class="button red-button" type="submit" name="Supprimer" value="Supprimer">
+                                    </form>
+                                </div>
+                                <br><br>
+                            </div>
+                        {% endfor %}
+                    </div>
+                {% endif %}
+                {% if all_active_club_memberships %}
```
Comment (TheoContentin) [inline back/social/templates/social/club_edit.html:149]: There is a problem in the logic here for the edge case where there is no longer any active member the buttons wont show. A variable all_inactive_club_membership  would be better

**Review by TheoContentin — COMMENTED** [review]
CR ok, execpt for the template of club edit. Very good job

**Review by LudoBike — APPROVED** [review]

**Review by sachabinder — APPROVED** [review]

**Pull request merged**